service/store/store.go

```diff
@@ -246,9 +246,6 @@ func validate(b Bid) error {
 	if err := b.AuctioneerID.Validate(); err != nil {
 		return fmt.Errorf("auctioneer id is not a valid peer id: %v", err)
 	}
-	if err := b.Sources.Validate(); err != nil {
```

Comment:
No longer validate Sources here.

Reply:
If this validation isn't here, should have a new place somewhere else in the code? I think might be in `SetAwaitingProposalCid` where gets persisted?

Reply:
The caller of `SetAwaitingProposalCid` in service.go does validation, but no harm to be more cautious. Will add